webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -223,15 +195,15 @@ function GitlabItems({item, theme}: GitlabItemsProps) {
             >
                 {item.action_name && (
                     <>
-                        <div>{item.updated_at && `${formatTimeSince(item.updated_at)} ago`}</div>
-                        <FormattedMessage
-                            id={notificationReasons[item.action_name].id}
-                            defaultMessage={notificationReasons[item.action_name].message}
-                        />
+                        <div>
+                            {item.updated_at && 'Updated'}
+                            {item.updated_at && `${formatTimeSince(item.updated_at)} ago`}
```

Comment:
```suggestion
                            {item.updated_at && `Updated ${formatTimeSince(item.updated_at)} ago`}
```

Reply:
https://github.com/mattermost/mattermost-plugin-gitlab/pull/300#discussion_r938274541
changed it due to this comment

Reply:
He has asked a question in that comment "Maybe this works correctly?". According to his comment, if the value for `item.created_at` and `username` are undefined then, it will print `Opened.`. Is this okay? I don't think so. You should tell him about this and revert the change that you did.